tests/test_beam_interface.py

```diff
@@ -351,4 +351,8 @@ def test_with_feeds_ordering_power(gaussian_uv):
     assert np.all(intf_feedx.polarization_array == [-5, -6, -7, -8])
 
     intf_feedyx = intf.with_feeds(["y", "x"], maintain_ordering=False)
-    assert np.all(intf_feedyx.polarization_array == [-6, -8, -7, -5])
+    # N.b. (Karto), this used to check against [-6, -8, -7, -5], but I _think_ this
+    # was actually a bug, in that UVBeam.select was sensitive to the ordering of
+    # pol arguments for polarization_array *only*, and not anything else with a
+    # polarization axis.
```

Comment:
Hmm. I need to dig into this more. This test isn't for UVBeams, it's for AnalyticBeams...

Reply:
Ah, then this I _think_ was a bug in UVBeam -- at issue is L3421 -> L3441 in UVBeam in the current verison of main. `final_pols` (which is assigned to `UVBeam.polarization_array` is set using `pol_inds` prior to a cal to sort, whereas what's in `data_array` is indexed _after_ a sort.